tests/test_processing.py

```diff
@@ -129,32 +129,34 @@ def test_calculate_buffer_size(
     )
 
 
-def test_draw_entropy_plot_error():
+def test_format_entropy_plot_error():
     with pytest.raises(TypeError):
-        draw_entropy_plot([])
+        format_entropy_plot(percentages=[], buffer_size=1024)
```

Comment:
I would also add a test case where we calculate the entropy of an empty file.

```python
p = Path("/tmp/empty")
p.touch()
report = calculate_entropy(p)
```

Reply:
I am not sure about testing exactly this bit (in fact I wanted to delete this test as well :)), as it looks a very corner-case, which is already handled by existing code at https://github.com/onekey-sec/unblob/blob/a2c396fdd765006336bade6648aa9e41a6ad7847/unblob/processing.py#L275 but some test will be added to check, that entropy is reported when processing a file.

Reply:
Makes sense.